client/src/components/ProductCard.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { CardTitle, Button, CardBody, Media } from 'reactstrap';
+import Product from './ProductFull';
+import Quantity from './Quantity';
+import Popup from './Popup';
+
+import './../styles/components/products/products-inner.css';	
+import './../styles/components/products/popup.css';	
+
+export default class ProductCard extends Component {
+	constructor(props) {
+		super(props)
+
+		this.state = {
+			displayProduct: false,
+			cartPopup: false,
+			qtyVal: 1
+		}
+
+		this.viewProduct = this.viewProduct.bind(this);
+		this.addToCart = this.addToCart.bind(this);		
+		this.getSelectedQty = this.getSelectedQty.bind(this);		
+	}
+
+	viewProduct() {
+		this.setState({
+			displayProduct: !this.state.displayProduct
```

Comment:
Use arrow functions when setting a value in the state in a way that relies on the previous state value